src/nemos/basis.py

```diff
@@ -204,6 +204,7 @@ def fit(self, X: FeatureMatrix, y=None):
         -------
         self :
             The transformer object.
+
```

Comment:
```suggestion
```
Surprised this isn't caught by ruff, but maybe because we're not including the docstrings?